clients/roscpp/package.xml

```diff
@@ -37,6 +40,9 @@
 
   <run_depend version_gte="0.3.17">cpp_common</run_depend>
   <run_depend>message_runtime</run_depend>
+  <run_depend>libboost-chrono-dev</run_depend>
+  <run_depend>libboost-filesystem-dev</run_depend>
+  <run_depend>libboost-system-dev</run_depend>
```

Comment:
Moving to format 2 would permit deduplicating these.

Reply:
Agreed moving to format 2 for all packages would be a nice improvement.

Regarding deduplication: in the context of [Generating ‘dev’ and runtime artefacts from ROS packages](https://discourse.ros.org/t/generating-dev-and-runtime-artefacts-from-ros-packages/12448) we're aiming for not using the `<depend>` tag to be able to generate pure runtime packages without bringing in all the dev dependencies.